h2o-py/h2o/h2o.py

```diff
@@ -2259,7 +2259,7 @@ def upload_mojo(mojo_path, model_id=None):
     """
     response = api("POST /3/PostFile", filename=mojo_path)
     frame_key = response["destination_frame"]
-    mojo_estimator = H2OGenericEstimator(model_key=get_frame(frame_key), path=mojo_path, model_id=model_id)
+    mojo_estimator = H2OGenericEstimator(model_key=get_frame(frame_key), model_id=model_id)
```

Comment:
what about emitting a warning if `model_id` is None? It would encourage user to set one + you could then print the backend-generated model_id after the upload.

Reply:
same can be done in R, in Flow can be done with backend warnings (?)

Reply:
yes, good points - this is for rel-zizler and it is a revert to the old behavior 

I don't assume people actually started relying on the new behavior - people upgraded for the log4j fixes

so my goal here is to get essentially remove this fix and have it 3.34.0.4 + log4j fixes

the model_id parameter I don't want to remove, so it stays